seed/challenges/html5-and-css.json

```diff
@@ -3407,7 +3407,7 @@
       ],
       "tests": [
         "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\", 'Give your <code>body</code> element the <code>background-color</code> of black.')",
-        "assert(editor.match(/<style>\\s*body\\s*\\{\\s*background-color\\s*:\\s*black;?\\s*\\}\\s*<\\/style>/i), 'Make sure your CSS rule is properly formatted with both and opening and closing curly bracket.')"
+        "assert(editor.match(/<style>\\s*body\\s*\\{\\s*background.*\\s*:\\s*.*;?\\s*\\}\\s*<\\/style>/i), 'Make sure your CSS rule is properly formatted with both and opening and closing curly bracket.')"
```

Comment:
Almost there. Please remove the first word `and` from `both and opening and closing curly`, and replace `background.*` with `background-color`, being more explicit is better. Also shouldn't here `bracket` be in plural?